build-tools/Dockerfile.rhel7.runtime

```diff
@@ -39,7 +39,7 @@ RUN microdnf --enablerepo=rhel-7-server-rpms --enablerepo=rhel-7-server-optional
     microdnf clean all
 
 COPY bigip-virtual-server_v*.json $APPPATH/vendor/src/f5/schemas/
-COPY as3-schema-3.25.0-3-cis.json $APPPATH/vendor/src/f5/schemas/
+COPY as3-schema-3.28.0-3-cis.json $APPPATH/vendor/src/f5/schemas/
```

Comment:
pattern match like in line 41 to avoid changes next time we upgrade AS3 version.
Applicable wherever relevant